app/src/main/java/com/example/cocktails/ui/main/ARFragment.kt

```diff
@@ -74,7 +93,7 @@ class ARFragment(context: Context) : Fragment() {
                 .glEsVersion
         if (openGlVersionString.toDouble() < MIN_OPENGL_VERSION) {
             Log.e(TAG, "Sceneform requires OpenGL ES 3.0 later")
-            Toast.makeText(activity, "Sceneform requires OpenGL ES 3.0 or later", Toast.LENGTH_LONG)
+            Toast.makeText(activity, "AR requires Android 4.3 or higher", Toast.LENGTH_LONG)
```

Comment:
@shahafhermann not sure AR is a common shortcut, maybe change to "Application requires Android 4.3 version or higher"? WDYT?

Reply:
Sure, I'll change it